timemachine/cpp/src/kernels/k_flat_bottom_bond.cuh

```diff
@@ -1,6 +1,65 @@
 #include "../fixed_point.hpp"
 #include "k_fixed_point.cuh"
 
+// branchless implementation of piecewise function
+template <typename RealType>
+RealType __device__ __forceinline__ compute_flat_bottom_energy(
+    RealType k, RealType r_lt_rmin, RealType r_gt_rmax, RealType r, RealType rmin, RealType rmax) {
+    return (k / 4) * (r_lt_rmin * (pow(r - rmin, 4)) + r_gt_rmax * (pow(r - rmax, 4)));
+}
+
+template <typename RealType>
+void __global__ k_log_probability_selection(
+    const int N,                             // Num atoms
+    const double kBt,                        // BOLTZ * temperature
```

Comment:
tiny nit: capital T for temperature? (`kBT`)

Reply:
Fixed in [ff96b2b](https://github.com/proteneer/timemachine/pull/915/commits/ff96b2b513c05b7cd3f1aec16f5550f71617f1bc)